Felienne/hedy
# [BUG FIX] Highscores are sorted correctly

**Description**
In this PR we fix the highscore ordering as we want to ordering on two dict values at the same time. The amount of achievements as well as the timestamp of the last achieved one.

**Fixes**
This PR fixes #2965.

**How to test**
Verify that the high scores are always sorted by amount of achievements (high-low) and then timestamp of last update (low-high). This ensures that the user with the most achievements and the lowest timestamp is ranked higher.

![Schermafbeelding 2022-06-27 om 09 39 52](https://user-images.githubusercontent.com/38155424/175886191-10b0af63-b6a6-4387-92df-dea1696b201d.png)
